synfig-studio/ui/shortcuts_window.glade

```diff
@@ -87,6 +87,11 @@
             </child>
           </object>
         </child>
+      </object>
+      <object class="GtkShortcutsSection">
```

Comment:
Shortcuts Window is broken

![Screenshot from 2020-11-16 20-03-51](https://user-images.githubusercontent.com/24214509/99265340-9a36be00-2847-11eb-8875-7bda2024851c.png)

Check the tags properly. Each sections should be wrapped inside `<child>` tags

Reply:
Also, don't use dash for shortcuts title property. We already have the main group titled **Canvas Zoom**
The rest can be titled as:
- Zoom in
- Zoom out
- Zoom fit

instead of
- Canvas-Zoom-In
- Canvas-Zoom-Out
- Canvas-Zoom-Fit

Reply:
@Keyikedalube Can you tell me why the travis build is failing? I always test in appveyor before pushing new changes. but now travis build failed. Kindly also tell me which is more genuine or better for synfig trial builds ?? Travis or appeveyor or CircleCI ?? 

Thanks :)

Reply:
Looking at Travis I don't think something's wrong with your code. You can see Autotools and CMake builds are fine. And that's what matters most because if either one fails then our commit has broken synfig. But it's not the case here.
It's OS X that's failing. Not related to code so no worry :)

Reply:
Resolved :)